osu.Game/Skinning/LegacySkinConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Game.Skinning
+{
+    public class LegacySkinConfiguration : DefaultSkinConfiguration
+    {
+        public const decimal LATEST_VERSION = 2.7m;
+
+        /// <summary>
+        /// Legacy version of this skin. Null if no version was set to allow fallback to a parent skin version.
```

Comment:
what parent?

Reply:
What about this?
```csharp
/// <summary>
/// Legacy version of this skin. Null if no version was set to allow <see cref="SkinProvidingContainer"/> fallback system.
/// </summary>

Reply:
At that point I don't even think it is necessary to mention this.